### PR TITLE
Add markdown to explain the logic to fil+ communities

### PR DESCRIPTION
**Pull request by xinaxu**

### CODE REVIEW

File: filplus.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Filplus Retrieval Sampling
+
+## Background
+
+The goal of filplus retrieval sampling is to offer an aggregated view of how each storage provider performs in terms of
+retrieval.
```
Comment (xmcai2016) [inline filplus.md:6]: "in terms of retrieval." -> "in retrievability"

File: filplus.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+It is a sampling of the retrieval performance of the storage providers in the Filecoin network.
+
+## Sampling Logic
+
+All active verified deals in Filecoin network will be sampled randomly.
+
+Newer deal will have a higher chance to be sampled than older deals. The chance is 4x for each year the deal is
```
Comment (xmcai2016) [inline filplus.md:13]: Newer deal -> Newer deals
The chance is 4x for each year the deal is newer -> The chance is 4x for each year the deal is newer determined by deal transaction date.

File: filplus.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+newer. This gives newer deal a higher chance to be sampled for retrieval testing.
+
+## Deployment
+
+Currently deployed in AWS Oregon, Frankfurt and Singapore regions. Retrievals will be performed from closest region to
```
Comment (xmcai2016) [inline filplus.md:18]: Let's also include AU's instance

**Review by xmcai2016 — APPROVED** [review]

File: filplus.md (hunk 3 above)
Comment (xinaxu) [reply filplus.md:18]: AU's instance is not running yet (and I'm not sure where is it running)

**Pull request merged**